Web/Edubase.Web.UI/Areas/Groups/Views/Group/CreateChildrensCentre.cshtml

```diff
@@ -288,13 +288,13 @@
                     <div id="children-centre-details" v-show="appState==='detail'">
                         <div class="govuk-grid-column-full">
 
-                            <h1 class="govuk-heading-xl" v-show="!pendingEdit">Enter the children's centre details</h1>
+                            <h1 id="enter-childrens-centre-details-heading" class="govuk-heading-xl" v-show="!pendingEdit">Enter the children's centre details</h1>
                             <h1 class="govuk-heading-xl" v-show="pendingEdit">Edit the children's centre group details</h1>
                         </div>
                         <div class="govuk-grid-column-full">
                             <p>Ensure this is the right children's centre and add a joined date.</p>
```

Comment:
I am a bit confused about what GitHub is trying to flag up with by highlighting the `>` characters throughout.

![image](https://user-images.githubusercontent.com/96429508/212327496-0dc8897b-f379-402d-838c-ce68000f9fe5.png)


It doesn't appear to be a whitespace change (e.g. a change in line endings) so I can only assume the GitHub syntax highlighting thinks there's an error in the original code (if so, we can presumably disregard the dark red highlighting of the `>` characters).

This PR / commit (eabce719): 
![image](https://user-images.githubusercontent.com/96429508/212327248-fc290987-bc48-4e99-a6a3-462a0eccad8c.png)

Previous commit (f0e958ab):
![image](https://user-images.githubusercontent.com/96429508/212327110-6a8d9a05-805e-444d-8a93-1489182a7b20.png)

Reply:
Looks like it's also present on `main` branch:

https://github.com/DFE-Digital/get-information-about-schools/blob/379336d601c0a9d2352747d3717262ea32831eb9/Web/Edubase.Web.UI/Areas/Groups/Views/Group/CreateChildrensCentre.cshtml#L288-L298


And the GitHub styling strongly suggests something the reason for highlighting is a syntax highlighting issue (speculation: maybe it doesn't work well with `cshtml` files?)

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/96429508/212329037-f52f0167-ade7-4f20-9a5f-12d82833bdd2.png">


Either way, doesn't seem to be a PR / code change problem, resolving this comment 👍